.eslintrc.js

```diff
@@ -28,7 +28,7 @@ module.exports = {
       parserOptions: {
         // The `ecmaVersion` should align to the supported features of our target runtimes (browsers / nodejs / others)
         // Consult with: https://kangax.github.io/compat-table/es2016plus/
-        ecmaVersion: 2017
+        ecmaVersion: 2018
```

Comment:
this is a JS option, for scripts / configuration which enables using `...` syntax